### ISSUE TITLE
[css-rhythm-1] Why is 'none' an alias to '0'?

### ISSUE THREAD

**Issue by kojiishi** [w3c/csswg-drafts#1151]
'`none`' was added in one of the spec review cycles, when '0' disables the rounding. Tab said we can define a value as _not negative_, but _positive_ is not possible, so we can't eliminate '0'.

I've got another feedback asking why we need '`none`', but I don't have answer.

Can we eliminate this, or is there any good explanation why this is needed?

@fantasai any thoughts?

**Comment by tabatkins** [comment]
You *can* eliminate the zero, just not syntactically - open ranges aren't allowed.  You're allowed to say that there's a minimum value (greater than zero, possibly UA-defined) that it's floored to.  We just don't like exposing to authors whether a particular UA rounds `.000001` differently from `.0000001`.

**Comment by fantasai** [comment]
There's no reason to floor the value. Rounding to the nearest 0-sized step *effectively* disables rounding, but mathematically it's continuous and consistent with just rounding to the nearest n-sized step. We added `none` because there was discussion of having line-height step take effect might have some behavior side-effects aside from simply rounding. E.g. it might affect `line-height: normal` calculation differently. (IIRC that was the context of adding `none`.)

**Comment by RByers** [comment]
@kojiishi, are you happy with @fantasai's answer?  Obviously if blink is going to ship, then we wont necessarily be able to change this later.

**Comment by kojiishi** [comment]
Sorry I was to respond to this but forgotten. Thank you Rick for the reminder.

So `none` is needed only when we add a feature like #938, that do something differently from just turning off, correct? I didn't understand that way when we added, sorry, but now it makes sense, thank you.

Currently #938 has i18n concern so I suppose it won't make as it is. Tab said he maybe come up with a new proposal that solves i18n concern, but we haven't got to it yet.

I think we can then:
* Remove `none` from the spec for now, since it's just an alias currently.
* When we've got an idea to add features that do differently from just turning off, add `none` back.
If UA supports `none`, changing its behavior is breaking, but if `none` is not parsed now, adding a new value later is easy, correct?

/cc @upsuper
